Doc 845 pagination examples (#2094)
n8n-io/n8n-docs: docs/code/cookbook/http-node/pagination.md

1. Set **Pagination Mode** to **Update a Parameter in Each Request**.
1. Set **Type** to **Query**.
1. Enter the **Name** of the query parameter. This depends on your API and is usually described in its documentation. For example, some APIs use a query parameter named `page` to set the page. So **Name** would be `page`.
1. Hover over **Value** and toggle **Expression** on.
1. Enter `{{ $pageCount + 1 }}`

`$pageCount` is the number of pages the HTTP Request node has fetched. It starts at zero. Most API pagination counts from one (the first page is page one). This means that adding `+1` to `$pageCount` means the node fetches page one on its first loop, page two on its second, and so on.

## Navigate pagination through body parameters

If the API you're using allows you to paginate through the body parameters:

1. Set the HTTP Request Method to **POST**
1. Set **Pagination Mode** to **Update a Parameter in Each Request**.
1. Select **Body** in the **Type** parameter.

> 1. Enter the **Name** of the body parameter. This depends on the API you are using. `page` is a common key name.

Reviewer: ```suggestion
1. Enter the **Name** of the body parameter. This depends on the API you're using. `page` is a common key name.
```
